debug/org.eclipse.cdt.debug.ui/src/org/eclipse/cdt/debug/ui/ICDebugUIConstants.java

```diff
@@ -98,4 +98,9 @@ public interface ICDebugUIConstants {
 	 */
 	public static final String CSOURCENOTFOUND_EDITOR_ID = PREFIX + "SourceNotFoundEditor"; //$NON-NLS-1$
 
+	/**
+	 * Debug 'instruction stepping mode' preference key.
+	 */
+	public static final String INSTRUCTION_STEPPING_MODE = "instruction_stepping_mode"; //$NON-NLS-1$
```

Comment:
Either remove this line, or at least change the value to be `ITargetProperties.PREF_INSTRUCTION_STEPPING_MODE` to avoid having the string in multiple locations.

Reply:
Fixed.